Lib/test/test_uuid.py

```diff
@@ -669,14 +667,41 @@ class TestUUIDWithExtModule(BaseTestUUID, unittest.TestCase):
 class BaseTestInternals:
     uuid = None
 
+    @unittest.skipUnless(_AIX, 'requires AIX')
+    def test_find_mac_netstat(self):
+        data = '''\
+Name  Mtu   Network     Address           Ipkts Ierrs    Opkts Oerrs  Coll
+en0   1500  link#2      fe.ad.c.1.23.4   1714807956     0 711348489     0     0
+                        01:00:5e:00:00:01
+en0   1500  192.168.129 x071             1714807956     0 711348489     0     0
+                        224.0.0.1
+en0   1500  192.168.90  x071             1714807956     0 711348489     0     0
+                        224.0.0.1
+'''
+        popen = unittest.mock.MagicMock()
+        popen.stdout = io.BytesIO(data.encode())
+
+        with unittest.mock.patch.object(shutil, 'which',
+                                        return_value='/usr/bin/netstat'):
+            with unittest.mock.patch.object(subprocess, 'Popen',
+                                            return_value=popen):
+                mac = self.uuid._find_mac_netstat(
+                    command='netstat',
+                    args='-ia',
+                    hw_identifiers=b'Address',
+                    get_index=lambda x: x,
+                )
+
+        self.assertEqual(mac, 0xfead0c012304)
```

Comment:
Assuming you go down the path of defining multiple subtests as described in 
https://github.com/python/cpython/pull/8672/files/70a45f01049079ddf00d3ba73f0fe3cf648d1985#r243934748, the expected results will need to be amended based on the value of `uuid._MAC_DELIM`: the AIX delimiter shouldn't report any results given the Mac OS X style `netstat` output, and vice-versa.

Reply:
I have re-done this so many times - I'll have to take some time to review your request. I'll act on it asap, but it will probably be days to weeks.

Many thanks for the review - Best wishes for 2019!

Reply:
Done in separate PR